### PR TITLE
Bugfix/defined but never used G ticket

### PR DESCRIPTION
**Pull request by artofmayhem**
Trello Card Defined but not used G - Group D FIXED

This FIXES the above stated Trello Card - 

HeadsUpDisplay/CountdownTimer.js
lines 2 and 3 component imports - imported for future usage. Commented out to relieve the error state and left comments regarding future usage on implementation of the feature. 

AccountSettings/RenderAccountSettings.js
line 2 Button element imported from antd but implemented as semantic button element resolved to antd Button element
line28: handleOk was assigned a value redundant with handleCancel. As both functions were for the handling of the modal visibility state I condensed it into a singular handleModal function that utilized the bang operator logic already baked into the code for simplicity and removing redundancy. 

Trello Card: https://trello.com/c/LckaYtEU
PR Video: https://youtu.be/mf9i9fgmLGs

-- CODE WAS REBASED BEFORE BRANCHING OFF NEW BUGFIX BRANCH --

Same test fail issue coming in from main noted

### CODE REVIEW

**Review by milklot — APPROVED** [review]
Look's good , less errors in the console.

File: src/components/pages/AccountSettings/RenderAccountSettings.js [hunk 1]
```diff
@@ -25,10 +25,7 @@ function RenderAccountSettings() {
   }, [user]);
 
   //These functions handle's exiting the modal once it is activated
-  const handleOk = () => {
-    setIsModalVisible(false);
-  };
-  const handleCancel = () => {
+  const handleModal = () => {
```
Comment (rhiannonstanford) [inline src/components/pages/AccountSettings/RenderAccountSettings.js:28]: Nice minor refactor to the code logic here.  It's more efficient and is more readable.  Handles setIsModalVisible() appropriately and the modal functions as expected.  Nice work!

File: src/components/pages/AccountSettings/RenderAccountSettings.js [hunk 2]
```diff
@@ -46,7 +43,7 @@ function RenderAccountSettings() {
     <div className="accountSettingsContainer">
       <Modal
         visible={isModalVisible}
-        onCancel={handleCancel}
+        onCancel={handleModal}
```
Comment (rhiannonstanford) [inline src/components/pages/AccountSettings/RenderAccountSettings.js:46]: This line is related to the revision above.  handleModal is a good name for this function.

File: src/components/pages/AccountSettings/RenderAccountSettings.js [hunk 3]
```diff
@@ -92,13 +89,13 @@ function RenderAccountSettings() {
             className="unlockButton"
             style={unlock ? null : { display: 'none' }}
           >
-            <button
+            <Button
```
Comment (rhiannonstanford) [inline src/components/pages/AccountSettings/RenderAccountSettings.js:92]: Where does this Button component come from? Ant Design library?  Is there an import statement?  Thank you!

File: src/components/pages/HeadsUpDisplay/CountdownTimer.js [hunk 4]
```diff
@@ -1,6 +1,8 @@
 import React, { useState, useEffect } from 'react';
-import { CurrentActivity } from './CurrentActivity';
-import { NextActivity } from './NextActivity';
+
+// TEMPORARY - Commenting out as this will be utilized once we render the implementation of the working activities display feature. Utilize components as necessary.
```
Comment (rhiannonstanford) [inline src/components/pages/HeadsUpDisplay/CountdownTimer.js:3]: Excellent use of comments to explain code context.

**Review by Yonathan-Admasu728 — APPROVED** [review]
clean fix and looks really concise. I love the inline comments. great job!

File: src/components/pages/AccountSettings/RenderAccountSettings.js (hunk 3 above)
Comment (ghost) [reply src/components/pages/AccountSettings/RenderAccountSettings.js:92]: There's an import for the Button from antd on line 2.

File: src/components/pages/AccountSettings/RenderAccountSettings.js (hunk 2 above)
Comment (ghost) [inline src/components/pages/AccountSettings/RenderAccountSettings.js:46]: Nice catch on the redundancy of the functions.

**Review by ghost — APPROVED** [review]
Good job using the built in Button feature from Ant Design and fixing the redundant functions for the modal!

**Pull request merged**